suite-native/transactions/src/components/TransactionDetail/TransactionDetailValuesSheet.tsx

```diff
@@ -0,0 +1,113 @@
+    handleSheetVisibility,
+    transaction,
+    fiatCurrency,
+}: TransactionDetailValuesSheetProps) => {
+    const { FiatAmountFormatter } = useFormatters();
+    const coins = useSelector(selectCoins);
+
+    const transactionInput = formatNetworkAmount(transaction.amount, transaction.symbol);
+    const transactionFee = formatNetworkAmount(transaction.fee, transaction.symbol);
+    const transactionTotal = formatNetworkAmount(
+        transaction.details.totalOutput,
+        transaction.symbol,
+    );
+
+    // historical rates
+    const fiatInputHistoryFormatted = FiatAmountFormatter.format(
+        toFiatCurrency(transactionInput, fiatCurrency.label, transaction.rates) ?? 0,
+    );
+    const fiatFeeHistoryFormatted = FiatAmountFormatter.format(
+        toFiatCurrency(transactionFee, fiatCurrency.label, transaction.rates) ?? 0,
+    );
+    const fiatTotalHistory =
+        toFiatCurrency(transactionTotal, fiatCurrency.label, transaction.rates) ?? 0;
+    const fiatTotalHistoryFormatted = FiatAmountFormatter.format(fiatTotalHistory);
+
+    // today rates
+    const actualRates = coins.find(coin => coin.symbol === transaction.symbol);
+    const fiatInputActualFormatted = FiatAmountFormatter.format(
+        toFiatCurrency(transactionInput, fiatCurrency.label, actualRates?.current?.rates) ?? 0,
+    );
+    const fiatFeeActualFormatted = FiatAmountFormatter.format(
+        toFiatCurrency(transactionFee, fiatCurrency.label, actualRates?.current?.rates) ?? 0,
+    );
+    const fiatTotalActual =
+        toFiatCurrency(transactionTotal, fiatCurrency.label, actualRates?.current?.rates) ?? 0;
+    const fiatTotalActualFormatted = FiatAmountFormatter.format(fiatTotalActual);
+
+    /**
+     * Creates Table header cell which contains percentual difference of transaction profit/loss.
+     */
+    const renderTodayHeaderCell = () => {
+        const fiatTotalHistoryNumeric = Number(fiatTotalHistory);
+        const fiatTotalActualNumeric = Number(fiatTotalActual);
+
+        const isProfit = fiatTotalHistoryNumeric < fiatTotalActualNumeric;
+        const renderPercentage = fiatTotalHistoryNumeric !== fiatTotalActualNumeric;
+        let percentageDifference;
+        if (isProfit) {
+            percentageDifference = Math.round(
+                (fiatTotalHistoryNumeric / 100) *
+                    (fiatTotalActualNumeric - fiatTotalHistoryNumeric),
+            );
+        } else {
+            percentageDifference = Math.round(
+                (fiatTotalActualNumeric / 100) * (fiatTotalHistoryNumeric - fiatTotalActualNumeric),
+            );
+        }
+
+        return (
+            <Text variant="hint" color="gray600">
+                Today{' '}
+                {renderPercentage && (
+                    <Text variant="hint" color={isProfit ? 'green' : 'red'}>
+                        {`${isProfit ? '+' : '-'}${percentageDifference}`}%
+                    </Text>
+                )}
+            </Text>
+        );
+    };
```

Comment:
This could be outside of the component to avoid creating function on every render